webapp/src/app.tsx

```diff
@@ -1755,10 +1767,11 @@ ${compileService && compileService.githubCorePackage && compileService.gittag ?
 
         return (
             <div id='root' className={rootClasses}>
-                {showExperimentalBanner ? <container.ExperimentalBanner parent={this} /> : undefined}
                 {hideMenuBar ? undefined :
                     <header className="menubar" role="banner">
                         {inEditor ? <accessibility.EditorAccessibilityMenu parent={this} highContrast={this.state.highContrast}/> : undefined }
+                        {showExperimentalBanner ? <notification.ExperimentalBanner parent={this} /> : undefined}
```

Comment:
this one too could be hidden into a "Notification" component thayt takes care of the logic of showing/hiding